files/ja/web/guide/user_input_methods/index.html

```diff
@@ -61,19 +61,19 @@ <h4 id="Mouse" name="Mouse">マウス</h4>
 
 <h4 id="Finger_touch" name="Finger_touch">指でのタッチ</h4>
 
-<p>When developing web applications meant to be installed on touchscreen devices, it’s a good practice to take into consideration the different capabilities in terms of screen resolution and user input. <a href="/ja/docs/Web/Guide/Events/Touch_events">Touch events</a> can help you implement interactive elements and common interaction gestures on touchscreen devices.</p>
+<p>タッチスクリーンデバイスにインストールされることを目的としている Web アプリを開発している時、ディスプレイ解像度とユーザー入力に関して異なる性能を考慮することは良いプラクティスです。<a href="/ja/docs/Web/API/Touch_events">タッチイベント</a>は、タッチスクリーンデバイス上のインタラクションな要素と一般的なインタラクションジェスチャーを実装するのに役立ちます。</p>
 
-<p>If you want to use touch events, you need to add event listeners and specify handler functions, which will be called when the event gets fired:</p>
+<p>タッチイベントを使いたいのであれば、イベントリスナーの追加とイベントが発火された時に呼び出されるハンドラー関数を指定する必要があります。</p>
```

Comment:
「イベントリスナーの追加」と「イベントが発火～ハンドラー関数を指定する」は同じことを言いなおしているだけなので、「と」で結ぶと意味が分かりにくくなります。下記のようにした方がいいと思います。
```suggestion
<p>タッチイベントを使いたいのであれば、イベントリスナーを追加して、イベントが発火された時に呼び出されるハンドラー関数を指定する必要があります。</p>
```

Reply:
ありがとうございます、こちら取り入れさせていただきます 🙏🏼